### ISSUE TITLE
Unhandled Fatal Exception

### ISSUE THREAD

**Issue by bk1984** [EDDiscovery/EDDiscovery#2577]
Running latest version of EDD.

```
An unhandled fatal exception has occurred.

==== BEGIN ====
code = Error (1), message = System.Data.SQLite.SQLiteException (0x800007BF): SQL logic error
no such table: Sectors
   at System.Data.SQLite.SQLite3.Prepare(SQLiteConnection cnn, String strSql, SQLiteStatement previous, UInt32 timeoutMS, String& strRemain)
   at System.Data.SQLite.SQLiteCommand.BuildNextCommand()
   at System.Data.SQLite.SQLiteDataReader.NextResult()
   at System.Data.SQLite.SQLiteDataReader..ctor(SQLiteCommand cmd, CommandBehavior behave)
   at System.Data.SQLite.SQLiteCommand.ExecuteReader(CommandBehavior behavior)
   at SQLLiteExtensions.SQLExtDataReader`1..ctor(DbCommand cmd, CommandBehavior behaviour, SQLExtTransaction`1 txn, SQLExtTransactionLock`1 txnlock) in C:\Code\EDDiscovery\BaseUtilities\SQLLite\SQLLite\SQLReader.cs:line 38
   at SQLLiteExtensions.SQLExtCommand`1.ExecuteDbDataReader(CommandBehavior behavior) in C:\Code\EDDiscovery\BaseUtilities\SQLLite\SQLLite\SQLCommand.cs:line 94
   at EliteDangerousCore.DB.SystemsDB.GetSystemListBySqDistancesFrom(SortedListDoubleDuplicate`1 distlist, Double x, Double y, Double z, Int32 maxitems, Double mindist, Double maxdist, Boolean spherical, SQLiteConnectionSystem cn, Action`1 LookedUp) in C:\Code\EDDiscovery\EliteDangerous\SystemDB\SystemsDBGetByDistance.cs:line 91
   at EliteDangerousCore.DB.SystemCache.GetSystemListBySqDistancesFrom(SortedListDoubleDuplicate`1 distlist, Double x, Double y, Double z, Int32 maxitems, Double mindist, Double maxdist, Boolean spherical, SQLiteConnectionSystem cn) in C:\Code\EDDiscovery\EliteDangerous\SystemDB\SystemCache.cs:line 172
   at EliteDangerousCore.StarDistanceComputer.BackgroundStardistWorkerThread() in C:\Code\EDDiscovery\EliteDangerous\EliteDangerous\StarDistanceComputer.cs:line 101
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
===== END =====
```

**Comment by turkwinif** [comment]
Running latest version as well. Crashes every few minutes ever since v11.0.1.0.
```
An unhandled fatal exception has occurred.

==== BEGIN ====
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at System.Collections.Generic.List`1.Add(T item)
   at EliteDangerousCore.DB.SystemsDB.GetSystemListBySqDistancesFrom(SortedListDoubleDuplicate`1 distlist, Double x, Double y, Double z, Int32 maxitems, Double mindist, Double maxdist, Boolean spherical, SQLiteConnectionSystem cn, Action`1 LookedUp) in C:\Code\EDDiscovery\EliteDangerous\SystemDB\SystemsDBGetByDistance.cs:line 106
   at EliteDangerousCore.DB.SystemCache.GetSystemListBySqDistancesFrom(SortedListDoubleDuplicate`1 distlist, Double x, Double y, Double z, Int32 maxitems, Double mindist, Double maxdist, Boolean spherical, SQLiteConnectionSystem cn) in C:\Code\EDDiscovery\EliteDangerous\SystemDB\SystemCache.cs:line 172
   at EliteDangerousCore.StarDistanceComputer.BackgroundStardistWorkerThread() in C:\Code\EDDiscovery\EliteDangerous\EliteDangerous\StarDistanceComputer.cs:line 101
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
===== END =====
```

**Comment by robbyxp1** [comment]
okay we will investigate

**Comment by klightspeed** [comment]
Want to Buy: separate post to new issue.
@turkwinif next time please create a new issue instead of hijacking an existing issue.  Your issue should be fixed by #2578

**Comment by robbyxp1** [comment]
This is an upgrade issue, let the upgrade complete before running the 3dmap.

thanks for the report.
